### ISSUE TITLE
[Intermediate][Deployment] Make rpm for OpenTenBase installation.

### ISSUE THREAD

**Issue by bartdong** [OpenTenBase/OpenTenBase#27]
Make rpm for OpenTenBase installation.

Goal: Make rpm for OpenTenBase installation.
Requirements: The local installation and testing must be completed accurately, and documents must be written and contributed to the community.

目标：制作一个OpenTenBase的rpm安装包。
要求：要在本地安装测试完成准确无误，同时编写文档贡献给社区

**Comment by James6xie** [comment]
Yes ，I noticed， opencloudos stream write rpm spec  and had build in koji  buildsystem：
https://gitee.com/opencloudos-stream/OpenTenBase/blob/master/OpenTenBase.spec
https://build.stream.opencloudos.tech/koji/buildinfo?buildID=6258

**Comment by James6xie** [comment]
and i build for opencloudos 8 and install ， there is conflict with libpq . 
if possible ,  I think opentenbase can required libpq instead of providing libpq itself.
